tools/runSchemaValidation.js

```diff
@@ -105,14 +105,14 @@ describe('run schema tests - ', () => {
         if (testResult.testsFailed > 0) {
 			let message = `there are failing schema test cases: `;
 
-			for (const testcase of testResult.testcases) {
-				if (!testcase.valid) {
-					message += "\n";
-					message += `\n     Test file: ${testcase.testFile}`;
-					message += `\n     Test name: ${testcase.testName}`;
-					message += `\n     ${testcase.message}`;
-				}
-			}
+			// for (const testcase of testResult.testcases) {
+			// 	if (!testcase.valid) {
+			// 		message += `\n`;
+			// 		message += `\n     Test file: ${testcase.testFile}`;
+			// 		message += `\n     Test name: ${testcase.testName}`;
+			// 		message += `\n     ${testcase.message}`;
+			// 	}
+			// }
```

Comment:
why are these  commented out?

Reply:
this is actually for the CI build of the PR. if you look at "All checks" at the bottom of your PR, you can get CI run for your PR. and you can see the result if you click on it. 

You can also run the test by running "npm test" in  "/tools" folder locally. 

---
In reply to: [245772133](https://github.com/Azure/azure-resource-manager-schemas/pull/542#discussion_r245772133) [](ancestors = 245772133)

Reply:
Sorry this was from my debugging. Will update in a minute